### PR TITLE
More SPI leftovers cleanup

### PR DESCRIPTION
**Pull request by mshaposhnik**
More SPI leftovers cleanup for KFXINFRA-660

### CODE REVIEW

File: components/sandbox/tiers/src/appstudio/ns_tenant.yaml [hunk 1]
```diff
@@ -120,18 +120,6 @@ objects:
   spec:
     hard:
       count/enterprisecontractpolicies.appstudio.redhat.com: "512"
-- apiVersion: v1
```
Comment (hugares) [inline components/sandbox/tiers/src/appstudio/ns_tenant.yaml:123]: When changing the tiers sources, we need to regenerate them for staging and prod, this is documented in the README.adoc in the tiers folder. The problem is we need to use a version of ksctl with an un-merged  change: https://github.com/kubesaw/ksctl/pull/50

I can regenerate the tiers and amend this PR if you want

Comment (hugares) [reply components/sandbox/tiers/src/appstudio/ns_tenant.yaml:123]: Done

**Review by hugares — APPROVED** [review]
/lgtm

File: components/sandbox/tiers/src/appstudio/ns_tenant.yaml (hunk 1 above)
Comment (mshaposhnik) [reply components/sandbox/tiers/src/appstudio/ns_tenant.yaml:123]: Thanks!

**Pull request merged**